### DISCUSSION TITLE
Partials and Trailing Newlines

### DISCUSSION THREAD

**Discussion by pvande** [mustache/spec#165] [Ideas]
Example:

``` mustache
{{! template.mustache }}
{{> partial }}
>> {{> partial }}
[ {{> partial }} ]

---
{{> special }}
>> {{> special }}
[ {{> special }} ]
```

``` mustache
{{! partial.mustache }}
{{! File ends with newline (common standard) }}
{{#things}}{{.}}, {{/things}}
```

``` mustache
{{! special.mustache }}
{{! File DOES NOT end with newline }}
{{#things}}{{.}}, {{/things}}
```

Currently specified output:

```
a, b, c, d, e, 
>> a, b, c, d, e, 

[ a, b, c, d, e, 
 ]

---
a, b, c, d, e, >> a, b, c, d, e, 
[ a, b, c, d, e,  ]
```

Proposed change:

Standalone partial tags should _always_ interpolate a trailing newline.
Non-standalone partial tags should _always_ chomp one trailing newline character before rendering.

Resulting output:

```
a, b, c, d, e, 
>> a, b, c, d, e, 
[ a, b, c, d, e,  ]

---
a, b, c, d, e, 
>> a, b, c, d, e, 
[ a, b, c, d, e,  ]
```

**Comment by jgonggrijp** [comment]
I think I'm against this. Users are in full control over what happens. The proposed change seems to be designed to give users neat-looking output, regardless of what they intended and how sloppy they were.
